docs/sc-tools/migrate_to_new_agent_api.md

```diff
@@ -19,9 +19,13 @@ Changed files include:
 
 Example of usage:
 ```shell
-./scripts/migration/migrate_to_new_agent_api.sh path/to/folder/with/project
+./scripts/migration/migrate_to_new_agent_api.sh path/to/folder/with/source/files
 ```
 
+!!! note
+    Pass path to source files that does not contain other projects as subfolders.
+    If path contains other projects as subfolders then those projects may be reformatted as well.
```

Comment:
I thinks it may be unclear for users.

I propose this variant:
`If there are subfolders with other projects in the folder at the specified path, sources of these projects will be modified as well.`

Reply:
tried to update note

Reply:
I wouldn't use the `reformat` term because reformatting refers to changing the layout of a text without any effect on its content. Maybe it had better to use the `change` (`fix`) term.